### ISSUE TITLE
Process details that should NOT be owned by W3M/Team/CEO

### ISSUE THREAD

**Issue by michaelchampion** [w3c/w3process#392]
As I told the AB yesterday, I'm happy with the text in https://w3c.github.io/w3process/director-free/ that assigns duties *that do not entail decisions* to the Team or CEO.  "The Team announces ...", "the Team provides ...", " the Team MUST monitor ...", "the Team organizes ...", etc.   

I'm also comfortable with text that gives the CEO  (delegating to the Team but being accountable for their actions) responsibility for making decisions about the implementation of policies and procedures:  "the CEO dismisses the participant for failing to meet the criteria", " the CEO determines whether the individual will be invited to participate ...", etc.   I suspect there are sections that say "Team"  but really should say "CEO" because a policy implementation decision is being made, but that's not critical given that the CEO formally runs the Team.

I agree with the suggestion to apply those changes soon (Process 2021?) to eliminate the numerous references to the Director that just cause confusion because the Team has always filled those roles and nobody argues they should not. 

But there are some strategic, technical, and/or policy-making decisions specified in the Process that need some Director surrogate to make or at least review and sign off on.  I'm VERY uncomfortable with the status quo where the nominal Director delegates these duties to W3M. True no disasters have happened on Ralph's watch, and he's done a good job channeling the spirit of Tim (and presumably asking for help on occasion), but the system is fragile and likely to fail when the next XHTML2/HTML5 or EME-scale breakdown in community consensus hits. (I can easily envision privacy or security issues that cause such a breakdown).

The specific problem areas I see in https://w3c.github.io/w3process/director-free/ that require broad community discussion and consensus building before putting in the Process include:

- Creating an overly large and inevitably STV-factionalized Council to adjudicate formal objections and other tie-breaking situations. 

- Creating a separate body to fill appointed TAG seats, which means neither body can effectively serve to counterbalance W3M.

- Giving the Team the authority to determine whether there is sufficient implementation experience.  That’s a strategic / technical judgment that the Team collectively isn’t independent enough to make given the vague criteria in the Process.  Some Director-surrogate should at least be on tap to give advice and consent on the hard cases.

- Giving the Team the authority to sign MoUs; this is a strategic decision that TimBL apparently got involved in fairly recently (e.g. the WHATWG MoU).  I hear stories of other SDOs / OSS projects seeking an MoU with W3C and got ignored by W3M.  Clearly the CEO can *negotiate* MoU's, but some Director surrogate (person or group) should guide the team on which organizations would be a good strategic partner and which working arrangements are consistent with W3C's principles.

- Giving the Team unfettered authority to propose new groups/charters to the AC.  We know that AC members very seldom give careful review to these proposals and the balloting is highly political, and that W3M creates new WGs for “business development” purposes to bring in new revenue little regard for whether they will create controversy later on.  This is a strategic decision that needs at least review from an independent body, maybe a strong Board of Directors that replaces the Hosts, maybe some sort of Director-surrogate.

This issue is about enumerating problems not proposing solutions.  But we might talk about constraints on / requirements for a "Director surrogate" role or roles.  I'd propose:

1. The way forward must have broad and early buy-in from the web standards community, not be cooked up in private by the AB/W3M and presented as a fait accompli to the AC.

2.  The "Director surrogate" roles should be concentrated in a single  group, strong enough to serve as a "check and balance" on W3M.  This is just a good governance principle, not a claim that W3M has abused the Benevolent Dictatorship role that the Director has delegated to them.

3.  The Director surrogate roles should focus on the *mission* not the *organization*.  The CEO and Team run the organization and are accountable for its financial stability and overall success.    Somebody else needs to be accountable for defining the mission (not the slogan, the concrete goals and operating principles) of W3C and assessing whether the organization is pursuing them effectively.  Some of these roles will eventually pass to an eventual Board of Directors to replace the Hosts, but much work needs to be done by hands-on web experts who communicate regularly, not merely governance/legal/finance specialists who meet occasionally.

**Comment by dwsinger** [comment]
Hi Mike, you open by expressing concern that the team/w3m has too much power, but then the first two bullets are places where we explicitly designed processes that do not give power to the team:

> Creating an overly large and inevitably STV-factionalized Council to adjudicate formal objections and other tie-breaking situations.
> 
> Creating a separate body to fill appointed TAG seats, which means neither body can effectively serve to counterbalance W3M.

This one, and chartering, I agree, we need to look at again. Not the mechanics of writing a charter, but deciding what to push through the process etc.; or the mechanics of verifying that there is an implementation report, but the judgment of whether it's enough:

> 
> Giving the Team the authority to determine whether there is sufficient implementation experience. That’s a strategic / technical judgment that the Team collectively isn’t independent enough to make given the vague criteria in the Process. Some Director-surrogate should at least be on tap to give advice and consent on the hard cases.

Why isn't the AC empowered to decide on whether the implementation evidence is adequate?

> Giving the Team the authority to sign MoUs; this is a strategic decision that TimBL apparently got involved in fairly recently (e.g. the WHATWG MoU). I hear stories of other SDOs / OSS projects seeking an MoU with W3C and got ignored by W3M. Clearly the CEO can negotiate MoU's, but some Director surrogate (person or group) should guide the team on which organizations would be a good strategic partner and which working arrangements are consistent with W3C's principles.

The intention was to assign MoUs to the Board, not the team. Did we miss?

**Comment by michaelchampion** [comment]
My concern with the first two bullets is that the Team retains de facto control since the alternative bodies are separate and inevitably weak

> Why isn't the AC empowered to decide on whether the implementation evidence is adequate?
The AC is empowered to, but has never done so in my experience.  In practice AC Review is a rubber stamp at the end of the process not a constructive input to the process

> The intention was to assign MoUs to the Board, not the team. Did we miss?
https://w3c.github.io/w3process/director-free/#Liaisons makes no mention of the Board, only the Team and CEO

**Comment by jeffjaffe** [comment]
>     * I hear stories of other SDOs / OSS projects seeking an MoU with W3C and got ignored by W3M.

@michaelchampion I would appreciate - perhaps privately - a pointer to where this has happened.  I'm not sure what you are referring to, but I am very concerned if this is true.

**Comment by ericsiow** [comment]
Proposed MOUs or any other agreements with external parties should be reviewed and approved by the Board.  The CEO can sign it but only after approval by the BoD.

**Comment by frivoal** [comment]
I agree that MoUs are generally a Board level matter. However:
* There isn't a clear distinction between an MoU and a any other contract, and we don't want to involve the Board in every single contract signing we go thought. Many contracts are trivial, some are momentous, and some fall in between. Regulating that ahead of time is not obvious.
* If the Board has the ultimate authority (which it should) it can take any matter it wishes into its own hands, and it can delegate anything as well. So generally, how much the board should do directly vs how much it should leave to the CEO (and the CEO's subordinates) should be up to the board to decide (and since the board should have the ability to fire the CEO if it is sufficiently displeased, it can enforce it), and we shouldn't force their hand one way or the other.

I do expect that a reasonable board should insist the MoUs like those with the WHATWG or earlier with IDPF are things that it ought to sign off on. But we don't need to write that down in the Process to make it true.

**Comment by dwsinger** [comment]
Council: we continue to feel it is the worst solution except for all the others

TAG appointment: we simplified

Assessing implementation experience: what charters can say and how to assess whether that has been met is a matter of active deliberation in the AC, but not per se a process issue.

MOUs: we insist that they get to the AC before it's too late; the Board may have more to say; the Bylaws will have legal requirements on certain matters.

Chartering: see separate discussions such as #328 #620 and #580

**Comment by frivoal** [comment]
As far as I can tell, every point in this issue has been addressed one way or another in https://www.w3.org/Consortium/Process/Drafts/director-free/, except for the question on new groups/charters, which is a duplicate of https://github.com/w3c/w3process/issues/580 (and discussed in a few other places as well). Proposing to close. If anything remains, please open individual focused issues.
